.github/workflows/linux_builds.yml

```diff
@@ -70,6 +70,23 @@ jobs:
         run: |
           ./bin/godot.linuxbsd.opt.tools.64 --test
 
+      # Download, unzip and setup SwiftShader library [d4550ab8d3f]
+      - name: Download SwiftShader
+        run: |
+          wget https://github.com/qarmin/gtk_library_store/releases/download/3.24.0/swiftshader.zip
+          unzip swiftshader.zip
+          rm swiftshader.zip
+          curr="$(pwd)/libvk_swiftshader.so"
+          sed -i "s|PATH_TO_CHANGE|$curr|" vk_swiftshader_icd.json
+
+      # Check class reference
+      - name: Check for class reference updates
+        run: |
+          echo "Running --doctool to see if this changes the public API without updating the documentation."
+          echo -e "If a diff is shown, it means that your code/doc changes are incomplete and you should update the class reference with --doctool.\n\n"
+          VK_ICD_FILENAMES=$(pwd)/vk_swiftshader_icd.json DRI_PRIME=0 xvfb-run bin/godot.linuxbsd.opt.tools.64 --doctool . 2>&1 > /dev/null || true
```

Comment:
`|| true` is not needed here.
This was only needed to validate memory leaks, because without it any leak e.g.
```
 Direct leak of 168 byte(s) in 1 object(s) allocated from:
    #0 0x7f138654a947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f1371ef00f3  (<unknown module>)
```
would break CI

Reply:
I'll try again but I was getting a non-zero return code without it.

Reply:
Sorry, I missed that `git diff` is needed after this command, so `|| true` can stay.